site/contents/write_ups/2022/16-06-22_writeup.md

```diff
@@ -0,0 +1,88 @@
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+
+```
+
+```{image} ../../../../images/AutomatingIneq.png
+:width: 200px
+:align: center
+:alt: Gru's Plan Meme, "Let the Algorithm decide exam grades" "People won't complain until it affects them" "Even Middle class people are affected". Meme made by Euan Bennet.
+```
+
+## Discussion
+
+As part of the Jean Goulding Institute's Data Week conference, we put on a special edition of Data Ethics Club, inviting our usual members as well as anyone attending Data Week to attend. Rather than give reading for the session beforehand, we watched a short video in the session. This video featured Virginia Eubanks talking about some of the ideas in her book (one our favourites here at DEC), Automating Inequality.
+
+In this brief piece, Eubanks brings up some of the more familiar ideas that regular DEC members will be familiar with about automating decision making. Additionally, she touched on ideas about flattening people into data and "maths-washing". We had less time than usual but endeavoured to discuss as many of these ideas as we could.
+
+## What are the pros and cons of automating decisions that affect people’s lives using algorithms compared to having humans make these decisions? 
+
+Pros: 
+In theory, automating decisions could reduce administrative load. Government programmes could *in theory* reduce resources spent on this stuff. Automation could reduce workload and streamline services. Take the example of student loan determination - deciding how much money students get. How much work would a single caseworker have to do for each student without some sort of automation? It's just bureaucracy in different forms. Humans reliably produce biases in problematic ways. If (BIG "if") automated decisions have carefully defined definitions of fairness then they could improve these decision making processes. Some things are easy to automate and can save a lot of people's time for more important things.
+
+Cons:
+Where do you set the boundaries for data inputs? Humans can widen inputs as they see fit.
+
+People object to disparity between the speed at which decisions are made about them, and the time it takes to object to a decision and appeal. Because "computer says no", but a human needs to take into account additional information that the computer cannot look for (without being told). If automated decisions are made, then there needs to be a streamlined process to object to decisions. This might make people feel a bit better about what's being done to them by automated systems. We felt that automatic the appeal process was *not* the appropriate measure here.
+
+The automated systems often ignore the point of what we're trying to achieve - a system might be designed to reduce government welfare spending rather than actually help people get the support they need. What you optimise for is an important variable to consider?
```

Comment:
```suggestion
The discussion about automated systems often ignores the point of what it is put in place to achieve - a system might be designed to reduce government welfare spending rather than actually help people get the support they need. What you optimise for is an important variable to consider.
```